pkg/util/githubutil/githubutil.go

```diff
@@ -159,3 +211,15 @@ func CheckBrewInstall(goos platformutil.Platform) (string, error) {
 	}
 	return "", nil
 }
+
+// GetSandboxImage Returns the alternate image if specified, else
+// if no version is specified then the Latest release of cr.flyte.org/flyteorg/flyte-sandbox:dind-{SHA} is used
+// else cr.flyte.org/flyteorg/flyte-sandbox:dind-{SHA}, where sha is derived from the version.
+func GetSandboxImage(version, image string) (string, string, error) {
```

Comment:
not : GetSandBoxImage -> GetFullyQualifiedImageName
pass in the bool flag of pre-release